tools/semtag

```diff
@@ -0,0 +1,625 @@
+#!/usr/bin/env bash
+
+PROG=semtag
```

Comment:
Probably worth mentioning the source repo we took this script from.

Reply:
Added https://github.com/terraform-aws-modules/terraform-aws-eks/pull/786/commits/6f910751246b9a25d6d3ccd16bf1961fa73ee1c6